src/utils.js

```diff
@@ -30,4 +30,35 @@ function getNodeProperty(node, path) {
   return property;
 }
 
-module.exports = getNodeProperty;
+/**
+ * Check if the file path follows an Embroider build pattern
+ * @param {string} filepath
+ * @returns boolean
+ */
+function hasEmbroiderPrefix(filepath) {
+  if (typeof filepath !== 'string') return;
+
+  const separator = filepath.includes('\\') ? '\\' : '/';
+  return filepath.split(separator).includes('embroider');
+}
+
+/**
+ * Get a file path with the Embroider prefix segments stripped out
+ * @param {string} filepath  E.g. /private/var/folders/abcdefg1234/T/embroider/098765/tests/acceptance/my-test.js
+ * @returns {string} E.g. tests/acceptance/my-test.js
+ */
+function getEmbroiderStrippedPrefixPath(filepath) {
+  if (typeof filepath !== 'string') return;
+
+  const separator = filepath.includes('\\') ? '\\' : '/';
+  const tokens = filepath.split(separator);
+
+  tokens.splice(0, tokens.lastIndexOf('embroider') + 2);
```

Comment:
Let's extract the `2` to a variable with a semantic name, something like `RELATIVE_PATH_ROOT`. Having magic numbers in the code makes it really hard for readers to understand what you're intent is, and they're left with inferring it.